chart/otomi/localtest-minikube.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
```

Comment:
The otomi chart does not seem to use this script.

Reply:
it is for devs only